Week1/homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+const bestDefender = ["Van Dijk", "Ramos", "Söyüncü"];
```

Comment:
I would add the `s` at the end of the variable, to show that there are more elements in the variable.

So I would change it to: `bestDefenders`

Reply:
> I would add the `s` at the end of the variable, to show that there are more elements in the variable.
> 
> So I would change it to: `bestDefenders`

yeah it is a quite logical : )